app/Services/VoucherEvaluator/Evaluations/ChildIsSchoolAge.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Services\VoucherEvaluator\Evaluations;
+
+use App\Specifications\IsBorn;
+use App\Specifications\IsUnderStartDate;
+use Carbon\Carbon;
+use Chalcedonyt\Specification\AndSpec;
+use Chalcedonyt\Specification\NotSpec;
+
+class ChildIsSchoolAge extends BaseChildEvaluation
+{
+    const REASON = 'school age (ineligible)';
+    private $specification;
+
+    public function __construct(Carbon $offsetDate = null, $value = 3)
+    {
+        parent::__construct($offsetDate, $value);
+
+        $this->specification = new AndSpec(
+            new IsBorn(),
```

Comment:
I think this is redundant - if a child is over school age they're also born - but not doing any harm.

Reply:
In this rule, that's true; we're never going to have a 5 year pregnancy. I'll mark it for removal at some point. In other rules it helps check for overdue pregnancies.